style.css

```diff
@@ -0,0 +1,207 @@
+html, body {
+	margin: 0;
+	padding: 0;
+}
+
+p {
+	margin: 0; 
+	padding: 0; 
+}
+
+.clearfix::after {
+  content: "";
+  display: table;
+  clear: both;
```

Comment:
Try to follow one style. If you use 2/4 spaces/tabs in your css, do it for each block of styles.